### ISSUE TITLE
Rubygems 1.7.x and compass gemspec

### ISSUE THREAD

**Issue by johnae** [Compass/compass#331]
The compass gemspec is invalid and rubygems 1.7.x will tell you that and won't let you run bundle install for example. I had to downgrade to rubygems 1.6.2 to make it work. This is very annoying and should probably be fixed in the gemspec. I guess the problem is this:

```
gemspec.date = "#{Time.now.year}-#{Time.now.month}-#{Time.now.day}"
```

which is not really valid - it should probably be

```
gemspec.date = Time.now.strftime('%Y-%m-%d')
```

**Comment by scottdavis** [comment]
is this change backwards compatible?

**Comment by scottdavis** [comment]
according to the gem manual date should be a unix timestamp integer?

<pre>
spec.date = File.utime('VERSION')
</pre>

**Comment by johnae** [comment]
Well if that's the case both the current gemspec and my suggestion are wrong and will probably result in the same behavior (i.e in my case not being able to run bundle install).

**Comment by scottdavis** [comment]
actually you shouldn't set this anyway
http://rubygems.rubyforge.org/rubygems-update/Gem/Specification.html#method-i-date-3D

**Comment by johnae** [comment]
Ahh ok, then that is the solution to this. Simply removing that from the gemspec should make everything good with rubygems 1.7.x.

**Comment by scottdavis** [comment]
fixed issue with gem spec in rubygems 1.7 closed by 73a01b367763d8bdc6d19b41b34e300bde67ae1a

**Comment by scottdavis** [comment]
this will get fixed in the beta.6 push
